<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>飞机大战</string>
  </property>
  <widget class="QLabel" name="background_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>507</width>
     <height>900</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="start_label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>510</y>
     <width>300</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="exit_label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>690</y>
     <width>300</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="title_label_1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>441</width>
     <height>96</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="title_label_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>290</y>
     <width>186</width>
     <height>38</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pause_label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>370</y>
     <width>300</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="sound_label">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>0</y>
     <width>60</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="copyright_label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>630</y>
     <width>300</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="help_label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>570</y>
     <width>300</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="helppage_label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>420</y>
     <width>350</width>
     <height>389</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="author_label">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>870</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>作者:liu@triumphal.cn</string>
   </property>
  </widget>
  <zorder>background_label</zorder>
  <zorder>helppage_label</zorder>
  <zorder>start_label</zorder>
  <zorder>exit_label</zorder>
  <zorder>title_label_1</zorder>
  <zorder>title_label_2</zorder>
  <zorder>pause_label</zorder>
  <zorder>sound_label</zorder>
  <zorder>copyright_label</zorder>
  <zorder>help_label</zorder>
  <zorder>author_label</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
